raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -3854,4 +3859,13 @@ private static ReplicaKey replicaKey(int id, boolean withDirectoryId) {
         Uuid directoryId = withDirectoryId ? Uuid.randomUuid() : ReplicaKey.NO_DIRECTORY_ID;
         return ReplicaKey.of(id, directoryId);
     }
+
+    private static Integer getRandomPort() {
+        int minPort = 1024;
+        int mockAddressPrefix = 9990;
+        // Number of nodes we can set up if we got a random number that is maximum in the range
+        int reservedNumberOfPorts = 50;
+        int maxPort = 65535 - mockAddressPrefix - reservedNumberOfPorts;
+        return ThreadLocalRandom.current().nextInt((maxPort - minPort) + 1) + minPort;
+    }
```

Comment:
Node ids are positive int32 including 0. Did you mean to name this method `randomReplicaId`?

Reply:
renamed and removed the minPort so it can generate from 0.